pkg/config_types.go

```diff
@@ -137,15 +138,115 @@ func (s SystemConfig) DatabaseConnection() (driverName, dataSourceName string) {
 }
 
 type ArtifactRepositoryS3Config struct {
-	KeyFormat       string
+	KeyFormat       string `yaml:"keyFormat"`
 	Bucket          string
 	Endpoint        string
 	Insecure        bool
 	Region          string
 	AccessKeySecret corev1.SecretKeySelector
 	SecretKeySecret corev1.SecretKeySelector
-	AccessKey       string
-	Secretkey       string
+	AccessKey       string `yaml:"accessKey"`
+	Secretkey       string `yaml:"secretKey"`
```

Comment:
should be `SecretKey`

Reply:
Do we need this if we have `ArtifactRepositoryS3Provider`? Can't we just move `AccessKey` and `SecretKey` fields to `ArtifactRepositoryS3Provider` and use that across the board?

Reply:
> should be `SecretKey`

Sorry, do you mean the `yaml: "secretKey"`?

Reply:
@aleksandrmelnikov the `K` in the actual field name is lower case now, should be uppercase.

Reply:
@rushtehrani I was able to move the attributes from S3Config to S3Provider.

Did you to remove the S3Config struct when you said "use across the board"?

Reply:
Made changes.